delivery_carrier_service_level/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_carrier_service_level,carrier.service.level,model_carrier_service_level,stock.group_stock_manager,1,1,1,1
```

Comment:
Stock managers can set service levels, but shouldn't normal stock users be able to at least read them, lest they get access errors?

Reply:
@thomaspaulb I think you are correct. Inventory user is not able to see the configuration but I think the should be able the read the service level on the stock picking

Reply:
But with your change now, he's also able to write it, eg. change the name of the service level.

Reply:
@thomaspaulb Ah you are correct. Fixed it to only read.